settings.php

```diff
@@ -58,14 +58,34 @@
         new moodle_url('/mod/customcert/download_all_certificates.php'), ''));
 }
 
+
+$settings->add(new admin_setting_heading('scheduledtaskconfig',
+    get_string('scheduledtaskconfigheading', 'customcert'),
+    get_string('scheduledtaskconfigdesc', 'customcert')));
+
+$settings->add(new admin_setting_configtext('customcert/certificatesperrun',
+    get_string('certificatesperrun', 'customcert'),
+    get_string('certificatesperrun_desc', 'customcert'),
+    0, PARAM_INT));
+$settings->add(new admin_setting_configcheckbox('customcert/includeinnotvisiblecourses',
```

Comment:
Again, I am not convinced this is needed. If the course is not visible to the user then they won't be able to view it anyways so why define it here?

Reply:
Before my edits, the plugin would check and process hidden courses. I changed this to a configurable setting because, in most cases, certificates are not issued for users in old, hidden courses. However, if there's a need to process everything, such as when a support ticket requires reissuing all certificates from previous terms, administrators still have the capability to do so. Does that make sense?

Reply:
Yep, making it a setting is the way to go! Thanks.